.gitignore

```diff
@@ -42,6 +42,8 @@ contrib/ucx
 bindings/java/pom.xml
 src/ucs/ucs_stats_parser
 test/gtest/gtest
+test/gtest/ucm/test_dlopen/.noinst/
+test/gtest/ucm/test_dlopen/rpath-subdir/.noinst/
```

Comment:
how were these files created? should we ignore them also in other directories?

Reply:
As I understand, these two directories were created because only two these makefiles set `noinst_libdir` variable for some purpose. Yeah, I think it may be useful to ignore them in other directories too.